external/stdk/README.md

```diff
@@ -0,0 +1,48 @@
+# Introduction
+
+The SmartThings Device SDK(for short STDK) is the core device library that allow device applications to securely connect to the SmartThings Cloud. 
+It has been integrated on TizenRT as a submodule. 
+
+## Prerequisites
```

Comment:
good to have documentation.
Please check md head level.  
And how about we also add this md file to repo's WiKi page

Reply:
how about let's upload this on confluence page first, provide internal test guide with it.